coalib/bearlib/languages/Language.py

```diff
@@ -423,7 +423,8 @@ def __or__(self, other):
     def __contains__(self, item):
         item = Language[item]
         return (type(self) is type(item)
```

Comment:
restructure all of this so that you can store `set(item.versions)` and `set(self.versions)` instead of building those sets multiple times.

Reply:
Still not done.